### ISSUE TITLE
Video playback not working in specific sites

### ISSUE THREAD

**Issue by arigit** [stoically/temporary-containers#107]
- Temporary Containers Version: 0.77
- Firefox Version: 59.0.1
- Operating System + Version: Linux, Fedora 27
- Non-default Options/Preferences in Temporary Containers: 
- Other installed Add-ons + Version: 

### Actual behavior

Open a temporary container tab.

Then load this URL: 
https://www.ole.com.ar/seleccion/movimientos-Messi-compania-Madrid_3_1984631525.html

The video does not play.
The same URL loaded in a regular tab works fine - the video plays normally.

Also, other Temporary Container tabs are able to play both flash and HTML5 video in this same system so it doesn't seem to be an issue with flash or video playing in general, but some specific problem with Temporary Container in this particular website.

### Expected behavior
video should play normally

### Steps to reproduce
as described above

### Notes
..

**Comment by stoically** [comment]
Hi, thanks for the report. Unfortunately I'm not able to reproduce this. Creating a fresh profile and loading the Website plays the Video correctly for me in a Normal Tab, Permanent Container Tab and Temporary Container Tab.

This is most likely an Add-on incompatbility. Can you please
* Provide the list of your Add-ons: You can navigate to `about:support` and Copy&Paste the list of "Extensions" there
* Try with a fresh profile and see if it happens for you just with the Temporary Containers Add-on?

Thanks!

**Comment by arigit** [comment]
I found a workaround to the problem - the below is for reference in case someone finds flash videos not working 

- Permanent Container, with my current firefox profile: not working

- Extensions (pretty standard)

Don't track me Google	4.21	true	dont-track-me-google@robwu.nl
GNOME Shell integration	10.1	true	chrome-gnome-shell@gnome.org
HTTPS Everywhere	2018.3.13	true	https-everywhere@eff.org
I don't care about cookies	2.8.1	true	jid1-KKzOGWgsW3Ao4Q@jetpack
Stylish - Custom themes for any website	3.0.3	true	{46551EC9-40F0-4e47-8E18-8E5CF550CFB8}
Temporary Containers	0.77	true	{c607c8df-14a7-4f28-894f-29e8722976af}
Text Contrast for Dark Themes	2.0.4	true	jid1-nMVE2oP40qeQDQ@jetpack
uBlock Origin	1.15.18	true	uBlock0@raymondhill.net

I tried disabling all extensions except for Temp Containers, restarted firefox, same problem.

Then I tested the URL in a regular container - it was no longer working. I know I tested this yesterday, and it was working. So not sure what's going on.

However, here's the workaround that worked for me: For this site, I disabled "Tracking Protection". I had it enabled by default. After doing this, video playback started to work withing all container types, normal and temporary. And with all Add-ons  turned. on.
